hydradx/model/amm/omnipool_amm.py

```diff
@@ -543,9 +543,10 @@ def lrna_swap(
         """
 
         if delta_qa < 0:
+            # selling LRNA
             asset_fee = self.asset_fee[tkn].compute()
-            if -delta_qa + self.lrna[tkn] <= 0:
-                return self.fail_transaction('insufficient lrna in pool', agent)
+            # if delta_qa + agent.holdings[tkn] <= 0:
+            #     return self.fail_transaction('Agent has insufficient lrna', agent)
```

Comment:
Are we doing this check somewhere else? What's the idea with this change?

Reply:
Put the check back in and fixed the affected tests.